plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1063,6 +1086,7 @@ def main():
             state=dict(default='present', choices=['present', 'absent', 'stopped', 'started', 'restarted', 'current']),
             storage=dict(type='str'),
             tablet=dict(type='bool'),
+            tags=dict(type='list', default=[""], elements='str'),
```

Comment:
```suggestion
            tags=dict(type='list', elements='str'),
```
I don't think there should be a default.